example/src/controllers/examples/subsections/BasicForm.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import { Form, Input, Errors } from 'react-formable';
+import FormData from './FormData';
+
+export default class BasicExample extends React.Component {
+
+    constructor(props) {
+        super(props);
+        this.state = {
+            formData: null
+        };
+    }
+
+    onChange(form) {
+        this.setState({
+            formData: form
+        })
+    }
+
+    onSubmit(form) {
+        if (form.valid)
+            alert('Tada!');
+    }
+
+    requiredValidator(value) {
+        if (!value)
+            return 'Required field missing!';
+    }
+
+    render() {
+        const inputStyles = {
+            marginLeft: '10'
+        }
+
+        return <div style={{ paddingTop: 15 }}>
+            <p>Here is a simple form. It's got a couple of fields and a bit of validation, easy right?</p>
```

Comment:
I don't know how this will work out, but lets try keeping all text to the `markdown` section of the subsections. I Imagine the structure of the subsections folder could be something like

```
examples/
  basic.md.js
  basic.form.js
  complex.md.js
  complex.form.js
```

When examples get complex and require multiple files... 

```
examples/
  basic.md.js
  basic.form.js
  complex/
    complex.md.js
    complex.form.js
    complex.input.js
```

Obviously filenames can change and this structure isn't solid yet, it's a direction to head to tho.